Makefile

```diff
@@ -204,7 +204,7 @@ endif
 		python:3 \
 		bash -c '/usr/local/bin/python release/get-contributors.py >> /code/AUTHORS.md'
 
-update-pins: update-go-build-pin
+update-pins: update-go-build-pin update-calico-base-pin
```

Comment:
Normally, we update calico/base together with calico/go-build before each release. This `update-pins` target will be triggered once we have a new calico/go-build release. Both calico/go-build and calico/base versions will be updated.